judge/domjudge-judgehost.service.in

```diff
@@ -7,9 +7,16 @@ After=network.target
 [Service]
 Type=simple
 
-ExecStart=@judgehost_bindir@/judgedaemon -n 0
+ExecStart=@judgehost_bindir@/judgedaemon
 User=@DOMJUDGE_USER@
 
+# If you want to run multiple judgedaemons per machine, or bind it to
```

Comment:
I prefer the stuff about multiple above (or a line explaining that its there) so people know its possible to do and than have the normal case.

And I found this:
https://www.stevenrombauts.be/2019/01/run-multiple-instances-of-the-same-systemd-unit/
Which makes it possible I think to use 1 file with multiple daemons. Not sure if it would make it easier for people who are not as experienced with linux.

Reply:
Both makes sense. I'm thinking of creating a second `domjudge-judgedaemon@.service` file, and just reference that from the first. The issue is that it requires extra users `domjudge-run-X` that are not created by default, and I prefer not to burden everyone with this.

Reply:
Just out of interest, not because I think its a very good solution:
Couldn't you use:
ExecStartPre to setup this needed user (make sure its there) assuming this is run as the root user, or use a type=oneshot (as root) which setsup the user. (https://www.freedesktop.org/software/systemd/man/systemd.service.html) I'm not really sure if it would be an improvement or actually just confusing.

Reply:
I think we had something like the `@.service` at past NWERCs and WFs, but can't find that script anymore. @nickygerritsen do you know where we have that?

Our recommendation is to pin the code to a single core, so we should make that easy, shouldn't we?

Reply:
I need to access that machine this week anyway so I will put the service file somewhere

Reply:
So the only differences:

* Rename the file to `domjudge-judgehost@.service`
* Change `-n 0` to `-n %i`

Then use `systemctl enable domjudge-judgehost@3` to enable on core 3.

Reply:
Jaap, do you agree that Nicky's proposal is the way to go?

Reply:
Yes. I was working on this, but while reorganizing the configure/makefile/... a bit, I dropped off a bit and this work is still sitting in my local branch.